File: ClientApp/src/components/Employee.js
import React, { Component } from 'react';

export class Employee extends Component {
    static displayName = Employee.name;

    render() {
        return (
            <div>
                <code>name, email, birthday, salary</code>
                <table>
                    <tr><td>Имя</td><td>значение</td></tr>
                    <tr><td>Почта</td><td>значение</td></tr>
                    <tr><td>Дата рождения</td><td>значение</td></tr>
                    <tr><td>Зарплата</td><td>значение</td></tr>
                </table>
            </div>
        );
    }
}

File: Models/Employee.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace my_new_app.Models
{
    public class Employee
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public DateTimeOffset Birthday { get; set; }
        [Column(TypeName="Money")]
        public decimal Salary { get; set; }
    }
}
